dlpy/model.py

```diff
@@ -2973,9 +2973,6 @@ class Optimizer(DLPyDict):
     bn_src_layer_warnings : bool, optional
         Turns warning on or off, if batch normalization source layer has
         an atypical type, activation, or include_bias setting. Default: False
-        freeze_layers_to : string
```

Comment:
so we dont' have descriptions for freeze_layers_to now. I am wondering if this is intentional

Reply:
It was documented twice.